### PR TITLE
fix package vulnerabilities in crypto-js

### PR DESCRIPTION
**Pull request by rameshvr**
fixes issue #165

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,12 +23,12 @@
     "test": "jest"
   },
   "dependencies": {
+    "buffer-equal-constant-time": "1.0.1",
+    "crypto-js": "^4.2.0",
```
Comment (asilde) [inline package.json:27]: The original issue mentions that [Crypto-js is no longer supported](https://www.npmjs.com/package/crypto-js). Is it possible to move to native Crypto without too much extra work?

Comment (rameshvr) [reply package.json:27]: Yes, have a draft PR on that. will push it after testing

**Comment by 2-towns** [comment]
Why do you need to keep crypto-js ? The signature check can be easily done with crypto from node.

**Comment by rameshvr** [comment]
> Why do you need to keep crypto-js ? The signature check can be easily done with crypto from node.

opened #167 to fix that. can you please review?

**Pull request merged**